direnvrc

```diff
@@ -60,6 +60,11 @@ use_nix() {
     unset TMPDIR
   fi
 
+  # `nix-shell --pure` sets IN_NIX_SHELL to pure which is
+  # misleading since we are in an impure shell now
+  # https://github.com/nix-community/nix-direnv/issues/28
+  export IN_NIX_SHELL=impure
```

Comment:
```suggestion
  # `nix-shell --pure` sets IN_NIX_SHELL to pure which is
  # misleading since we are in an impure shell now
  export IN_NIX_SHELL=impure
```